week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -15,12 +15,30 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 */
 
 function sortAges(arr) {
+ let maxarray = [];
+ for ( let i = 0 ; i <= arr.length ; ++i ){
+   if (typeof arr[i] == `number`){
+     maxarray.push(arr[i]);
+   }
+ }
+ sortAge(maxarray);
+ console.log (maxarray);
+ return maxarray;
+}
+// for sort number 
+
+function sortNumber(a, b) {
+  return a - b;
+}
 
+function sortAge(maxarray){
+ maxarray.sort(sortNumber);

```

Comment:
If you read the instructions carefully it says that you aren't supposed to use the `.sort()` array method.

However, this is a very tricky exercise! So well done for getting everything else correct.